src/PluginForm/OpenInvoicePaymentForm.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace Drupal\commerce_adyen\PluginForm;
+
+use Drupal\commerce\Response\NeedsRedirectException;
+use Drupal\commerce_adyen\Adyen\Authorisation\Request;
+use Drupal\commerce_payment\PluginForm\PaymentOffsiteForm as BasePaymentOffsiteForm;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+use germanoricardi\helpers\BrazilianHelper;
+
+/**
+ * OpenInvoice Form.
+ */
+class OpenInvoicePaymentForm extends BasePaymentOffsiteForm {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $form = parent::buildConfigurationForm($form, $form_state);
+
+    $form['gender'] = [
+      '#type' => 'radios',
+      '#title' => t('Gender'),
+      '#options' => [
+        'MALE' => t('Male'),
```

Comment:
s/t/$this->t()

Reply:
It is because currently extended forms not let $this->t('...') please view that I report now this. https://www.drupal.org/project/commerce/issues/3031669